corehq/motech/generic_inbound/views.py

```diff
@@ -214,3 +225,38 @@ def _execute_case_api(domain, couch_user, device_id, context, api_model):
         'form_id': xform.form_id,
         'case': serialize_case(case_or_cases),
     }
+
+
+def _log_api_request(api, request, response):
+    is_success = response.status_code == 200
+    status = RequestLog.Status.SUCCESS if is_success else RequestLog.Status.ERROR
+    response_body = response.content.decode('utf-8')
+    log = RequestLog.objects.create(
+        domain=request.domain,
+        api=api,
+        status=status,
+        response_status=response.status_code,
+        error_message=response_body if not is_success else '',
+        username=request.couch_user.username,
+        request_method=request.method,
+        request_query=request.get_raw_uri(),
```

Comment:
Looking at this again, I wonder if it should use `request.META.get("QUERY_STRING")`, since I don't think there's anything useful before the `?`

Reply:
Yup, agree